charts/intel-gaudi-resource-driver/values.yaml

```
@@ -19,13 +19,27 @@ serviceAccount:
 
 kubeletPlugin:
   podAnnotations: {}
+  nodeSelector: {}
+  # label used when nfd.enabled is true
+    #intel.feature.node.kubernetes.io/gaudi: "true"
```

Comment could clearer above the item it's referring to:
```suggestion
  # nodeSelector specified here is used only when nfd.enabled is false
  nodeSelector: {}
```